src/bundle/Resources/public/scss/fieldType/edit/_ezcontentquery.scss

```diff
@@ -0,0 +1,19 @@
+.ez-field-edit--eznoneditable {
```

Comment:
This should not be in `_ezcontentquery.scss`
this is a generic class, not related to any fieldtype